.ci.yaml

```diff
@@ -500,7 +519,6 @@ targets:
     recipe: engine_v2/engine_v2
     timeout: 60
     properties:
-      release_build: "true"
```

Comment:
From the PR description:

> Release_build property is used to signal that this build produces release artifacts that will eventually be SLSA compliant and will need to run from dart_internal.

I didn't quite follow why we need to remove here. Do we not want to run them against dart_internal any more?

Reply:
Only builders generating artifacts will be running in dart-internal. Builders like "Mac impeller-cmake-example" are test only and should not run in dart-internal.